### PR TITLE
Feature oshc

### PR DESCRIPTION
**Pull request by jen-p**

### CODE REVIEW

**Comment by craigmcginley** [comment]
Deploy preview for *loving-austin-5f2eff* ready!

Built with commit 298d7f1ce4d66506e63ff6f3a32bb7d53da98d62

https://deploy-preview-164--loving-austin-5f2eff.netlify.com

File: src/pages/vision/open-source-healthcare/index.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react'
+
+import Layout from '../../../components/layouts/layout'
+import Hero from '../../../components/hero'
+import Image from '../../../components/image'
+import Divider from '../../../components/divider'
+//import References from '../../../components/references'
```
Comment (craigmcginley) [inline src/pages/vision/open-source-healthcare/index.js]: Can delete this

File: src/pages/vision/open-source-healthcare/index.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+              </div>
+
+              <div className="oshc-introduction">
+                <h4 className="header--sm margin-bottom--none">We have open standards for finance</h4>
+                <p className="text--gray margin-top--none margin-bottom--double">because we value our money more than our health.</p>
+
+                <h4 className="header--sm margin-bottom--none">We have open standards for transportation</h4>
+                <p className="text--gray margin-top--none margin-bottom--double">because getting to your destination is a necessity.</p>
+
+                <h4 className="header--sm margin-bottom--none">We have open standards for healthcare</h4>
```
Comment (craigmcginley) [inline src/pages/vision/open-source-healthcare/index.js]: *"need" open standards for healthcare

File: src/pages/vision/open-source-healthcare/index.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+                <p className="text--gray margin-top--none margin-bottom--double">because our lives depend on it.</p>
+
+                <p className="text--gray">Read our open source ethos,
+                  <br />
with contributing articles by <a href="https://www.healthpopuli.com/2018/10/11/open-source-health-care-will-liberate-patients/" target="_blank" rel="noopener noreferrer">Jane Sarasohn-Kahn</a> and <a href="https://twitter.com/EricTopol" target="_blank" rel="noopener noreferrer">Eric Topol</a></p>
+
+                  <div className="button-group margin-bottom--double">
+                    <div className="pure-u-1 pure-u-lg-1-2"><a href="" target="_blank" rel="noopener noreferrer" className="button button--primary button--lg margin-top--double margin-bottom--double button--block">Download 25 MB PDF</a></div>
```
Comment (craigmcginley) [inline src/pages/vision/open-source-healthcare/index.js]: Needs a real link

File: src/pages/vision/open-source-healthcare/index.js [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+                    <div className="pure-u-1 pure-u-lg-1-2"><a href="" target="_blank" rel="noopener noreferrer" className="button button--primary button--lg margin-top--double margin-bottom--double button--block">$12 Blurb Magazine</a></div>
```
Comment (craigmcginley) [inline src/pages/vision/open-source-healthcare/index.js]: Needs a real link

**Comment by jen-p** [comment]
pushed changes for work and feature linking, typo fix, and button links

**Comment by craigmcginley** [comment]
Pushed one more tweak to use Gatsby's internal Link component for the work page OSHC link. I'm good on this now!

**Pull request merged**